draft-packages/illustration/KaizenDraft/Illustration/Base.tsx

```diff
@@ -20,20 +20,48 @@ export type BaseProps = {
    * @default ""
    */
   classNameAndIHaveSpokenToDST?: string
+
+  /**
+   * Aspect ratio that is set on the illustration in Scene/Spot which wraps the
+   * component in a container, forcing the aspect ratio.
+   */
+  aspectRatio?: "landscape" | "portrait" | "square"
 }
 
 export const Base = ({
   name,
   alt,
   classNameAndIHaveSpokenToDST,
+  aspectRatio,
   ...otherProps
 }: BaseProps) => {
   const className =
     (classNameAndIHaveSpokenToDST ? classNameAndIHaveSpokenToDST : "") +
     " " +
     styles.wrapper
 
+  const aspectClassName =
+    (aspectRatio ? styles[aspectRatio] : "") + " " + styles.aspectRatioWrapper
+
   return (
-    <img {...otherProps} className={className} alt={alt} src={assetUrl(name)} />
+    <>
```

Comment:
Would be nice to add some sort of unit test for this, as it changes what it renders depending on the `aspectRatio` prop.

Reply:
Yeah good shout. Will add one that checks the class.